sycl/source/detail/graph_impl.hpp

```diff
@@ -331,6 +331,24 @@ class node_impl {
     }
   }
 
+  /// Test is the node contains a N-D copy
+  /// @return true is the op is a N-D copy
+  bool is2DCopyNode() const {
```

Comment:
Should we rename this to `isNDCopyNode` to reflect that we check for dimensions larger one?